### DISCUSSION TITLE
How to use WebView to inject JavaScript and use Arturo.js in a web application?

### DISCUSSION THREAD

**Discussion by scifx** [arturo-lang/arturo#1778] [Q&A]
I have observed that Arturo requires improvement in GUI design. 

Although there is already a "**webview**" component available, I am struggling to use the **`.inject`** method to embed JavaScript into the WebView and haven't had much success doing so.

 It appears that there is limited documentation demonstrating how to develop web applications using **Arturo.js**. 
I am closely following the progress of Arturo's GUI development and currently find "**arturo.js**" confusing, suspecting it may hold key to Arturo's GUI capabilities. 

Due to the lack of relevant documentation, I ultimately had to rely on **Python** and libraries like **gradio** and **pywebview** to achieve similar effects.

I hope that future updates will provide more examples demonstrating the use of "**webview**" and "**arturo.js**" for developing desktop or web applications, helping me better understand and utilize these tools.

Thank you for your work!

**Comment by Eloitor** [comment]
Some examples or using .inject:

- https://mermaid.pkgr.art/
- https://mathjax.pkgr.art/
